website/docs/configuration/expressions/references.html.md

```diff
@@ -110,6 +174,12 @@ _temporary variables_ in their documentation. These are not [input
 variables](/docs/configuration/variables.html); they are just arbitrary names
 that temporarily represent a value.
 
+The names in this section relate to top-level configuration blocks only.
+If you use [`dynamic` blocks](dynamic-blocks.html) to dynamically generate
+resource-type-pecific _nested_ blocks within `resource` and `data` blocks then
+you'll the key and value of each element differently. See the `dynamic` blocks
```

Comment:
> you'll the key and value of each element differently.
I'm not quite sure what this means? You'll need to obtain the key and value differently, maybe?

Reply:
It seems like I made a number of editorial errors here that I missed on my final review. Also see "pecific" instead of "specific" on the previous line. I'll revise this to make it actually make sense. :grinning: